### PR TITLE
Refactored to smaller components

### PR DESCRIPTION
**Pull request by Doodlis**

### CODE REVIEW

File: src/components/OverviewComponent/Overview.js [hunk 1]
```diff
@@ -51,177 +45,45 @@ const Overview = (props) => {
...
 
+    setSelectedSize(event.item.props.skuId);
+    setSelectedSizeLetters(event.item.props.value);
+  };
 
- /// work in progres for handleQuantityClick
+  /// work in progres for handleQuantityClick
   const handleQuantityClick = (e) => {
     message.info('Click on menu item.');
     console.log('click', e);
```
Comment (jessepye) [inline src/components/OverviewComponent/Overview.js:60]: console.log that could be removed for the main branch

File: src/components/OverviewComponent/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    infinite: false,
+    speed: 500,
+    initialSlide: 0,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+  };
+
+
+  return (
+    <>
```
Comment (jessepye) [inline src/components/OverviewComponent/Carousel.js:38]: Are the fragments necessary? (I don't actually know) Doesn't the 'carousel-place-holder' div already contain everything?

**Review by jessepye — CHANGES_REQUESTED** [review]
Looks good! I only found one stray console.log and one stray <></> fragment

File: src/components/OverviewComponent/Carousel.js (hunk 2 above)
Comment (Doodlis) [reply src/components/OverviewComponent/Carousel.js:38]: yes

File: src/components/OverviewComponent/Overview.js (hunk 1 above)
Comment (Doodlis) [reply src/components/OverviewComponent/Overview.js:60]: still work in progress, I would remove it here just to add it back in

**Pull request merged**